src/validators/email.js

```diff
@@ -2,5 +2,5 @@ import i18n from '@dhis2/d2-i18n'
 
 export const emailPattern = /^.+@.+\.[a-zA-Z]+$/
 export const invalidEmailMessage = i18n.t('Not a valid e-mail')
-export const validEmail = value =>
-    emailPattern.test(value) ? undefined : invalidEmailMessage
+export const email = value =>
+    value && !emailPattern.test(value) ? invalidEmailMessage : undefined
```

Comment:
You could keep the ternary check as is here right?

Reply:
No, this function should not return an error for an empty string which doesn't match the regex pattern. So these changes were justified.

Reply:
No I mean that the ternary check has been reversed. I'm mentioning it because negation always seem slightly harder to read. It's functionally the same so just a style remark.